src/aws-cpp-sdk-core/include/smithy/identity/resolver/AwsBearerTokenIdentityResolver.h

```diff
@@ -4,16 +4,92 @@
  */
 #pragma once
 
+#include <aws/core/auth/bearer-token-provider/AWSBearerTokenProviderBase.h>
+#include <aws/core/auth/bearer-token-provider/SSOBearerTokenProvider.h>
+#include <smithy/identity/identity/AwsBearerTokenIdentity.h>
 #include <smithy/identity/resolver/AwsIdentityResolverBase.h>
 
-#include <smithy/identity/identity/AwsBearerTokenIdentity.h>
+namespace smithy
+{
+
+static const char SSO_DEFAULT_BEARER_TOKEN_PROVIDER_CHAIN_LOG_TAG[] =
```

Comment:
same comment as above, maybe we should just inline these to avoid pollution

Reply:
I can put it in the class scope to make it better